core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -112,7 +112,7 @@ class PluginTypeChecker(platform: String = "onprem") extends Logging {
     // Some SQL function names have backquotes(`) around their names,
     // so we remove them before saving.
     readSupportedOperators(source, "exprs").map(
-      x => (x._1.toLowerCase.replaceAll("\\`", ""), x._2))
+      x => (x._1.toLowerCase.replaceAll("\\`", "").replaceAll(" ",""), x._2))
```

Comment:
Just in case, is it possible to have "\\t" too? then we would replace all white_spaces?
It is not specific to this PR because we have the same hing all over the place.

Reply:
I checked all the expressions and don't see "\t" . They are separated by whitespaces. So I think it's okay to leave it as it is.

Reply:
Yeah, here it is checking a CSV file from the repo which is controlled internally.
I meant the "\t" that may come from an eventLog if any. For example, the SQL query has "\t" character as prefix/postfix of an expression.
but anyway I am okay with leaving it as is if the white-space is not likely to be an issue